src/components/course/course-header/deprecated/CourseRunCard.jsx

```diff
@@ -49,6 +49,13 @@ const CourseRunCard = ({
     isEnrollable,
   } = courseRun;
 
+  const intl = useIntl();
+  const DEFAULT_BUTTON_LABEL = intl.formatMessage({
+    id: 'enterprise.course.about.page.course.run.card.enroll.button.label',
+    defaultMessage: 'Enroll',
+    description: 'Default button label for enrolling in a course run',
```

Comment:
Can we add little bit more context? Will this button redirect the user to some page? what will happen when clicked?